lib/xcodeproj/workspace/group_reference.rb

```diff
@@ -0,0 +1,64 @@
+module Xcodeproj
+  class Workspace
+    # Describes a group reference of a Workspace.
+    #
+    class GroupReference
+      # @return [String] the name of the group
+      #
+      attr_reader :name
+
+      # @return [String] the type of reference to the project
+      #
+      # This can be of the following values:
+      # - absolute
+      # - group
+      # - container (only supported value)
```

Comment:
Should we throw in case we encounter a non-supported value, or just silently fail? (\cc @segiddins)